static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -124,15 +131,26 @@ function ConsoleMessage({
   );
 }
 
-const Common = styled('div')<{isLast: boolean; level: string}>`
+const Common = styled('div')<{
+  isLast: boolean;
+  level: string;
+  isEmitted?: boolean;
```

Comment:
Let's get some other feedback on this but we may want to apply this to timestamp too. you'll need to be careful with the tooltip.

Reply:
This is how it would look like! 
![image](https://user-images.githubusercontent.com/39612839/175574337-7bd7ec81-92da-4438-a238-cd8ff48b8886.png)